index.js

```diff
@@ -11,3 +11,39 @@ mongoose
   })
   .then(x => console.log(`Connected to Mongo! Database name: "${x.connections[0].name}"`))
   .catch(err => console.error('Error connecting to mongo', err));
+
+Recipe.create({ 
+  title: 'Carrot cake',
```

Comment:
Explaining a little bit about Promisse:
```js
- Recipe.create (newRecipe);
- Recipe.insertMany (data);
- Recipe.updateOne ({title: 'Rigatoni alla Genovese'}, {duration: 100});
-Recipe.deleteOne ({title: 'Carrot Cake'});
```

All of them are promises and they are only executed in 2 situations: when you call the "then" after the promise, or using Promisse.all

The first form, you would do as you did:
```
Recipe.create ({
  title: 'Carrot cake',
  cuisine: 'Meh',
  dishType: 'Breakfast'
})
.then (recipe => console.log (recipe.title)) <= promisse executed here
.catch (error => console.log (error));
```


If you want to execute more then 1 promisse sequentially:

``` js
Recipe.create ({
  title: 'Carrot cake',
  cuisine: 'Meh',
  dishType: 'Breakfast'
})
.then (recipe => console.log (recipe.title))
.then (() => Recipe.updateOne ({title: 'Rigatoni alla Genovese'}, {duration: 100})
.then ((recipe) => {
    console.log ('now it is good', recipe))
   return Recipe.deleteOne ({title: 'Ratatouille'})}
.then ((recipe) => {
   console.log (recipe.title)
 return mongoose.connection.close ()
}).then (() => console.log (`connection closed`))
.catch (err => console.log (`an error has occurred: $ {err}`));
```

Another way, using Promisse All:

```js
let recipeUpdate = Recipe.updateOne ({title: 'Rigatoni alla Genovese'}, {duration: 100});
let recipeRemove = Recipe.deleteOne ({title: 'Carrot Cake'});

Promisse.all ([recipeUpdate, recipeRemove])
.then (result => {
    console.log (`The recipe is updated - the recipe is deleted`);
    return mongoose.connection.close ()}
.then (() => console.log (`connection closed`))
.catch (err => console.log (`an error has occurred: $ {err}`));
```
👏🏻Congrats to finish the assignment